reference/release-notes/1.24.0.md

```diff
@@ -0,0 +1,96 @@
+
+### Updates to components
+
+* Anbox Cloud includes and uses the following latest versions for components it uses:
+  - LXC version 6.0.2<!--AC-2812-->
+  - Mesa driver version 24.2.6<!--AC-2800-->
+  - VirGL renderer version 1.9.0.<!--AC-2800-->
+
+### Security
+
+* We have enhanced security with a minimum version of TLS 1.3 as a requirement for all services working with Anbox Cloud. TLS 1.3 is also enabled by default for the Anbox Stream Agent and Gateway.<!--AC-2728 and AC-2654-->
+
+### Anbox Management Service (AMS)
+* We have increased the maximum number of queued operations in AMS to allow better performance.<!--AC-2831-->
+* Anbox Cloud provides the image type (`android`, `aaos` or `generic`) through the following three ways:<!--AC-2757-->
+  - `metadata.yaml` of the Anbox Cloud image
+  - API response from the AMS when querying image information
+  - Output of `amc image show` command
+
+  This allows the web client to adapt its interface according to the image type.
+* We have enabled support for multiple cluster management by AMS.<!--AC-2674-->
+
+### Android
+
+* This release includes Android 14 based AOSP images.
+* We now support using the lock screen in Anbox Cloud([LP 1983426](https://bugs.launchpad.net/anbox-cloud/+bug/1983426)).
+
+### Charms
+
+* For the NFS Operator charm for Anbox Cloud, `Cachefilesd` is now optional and is used only when `fsc` is provided as a mount option.<!--AC-2839-->
+
+### Streaming improvements
+
+* Anbox Cloud now provides an option to allow the use of custom fragment shader-based upscaling algorithms for video streaming. This helps you to apply multiple shaders sequentially to perform multi-pass upscaling and replaces the default [AMD FidelityFX Super Resolution 1.0](https://gpuopen.com/fidelityfx-superresolution/) shader. See {ref}`sec-custom-fragment-shader` for more information.<!--AC-2541-->
+* The Anbox Streaming SDK now uses the [Snapdragon™ Game Super Resolution(SGSR)](https://github.com/SnapdragonStudios/snapdragon-gsr) algorithm, which integrates upscaling and sharpening in one single GPU shader pass, by default.<!--AC-2543-->
+
+### Other
+
+* Android security updates for November 2024 (see [Android Security Bulletin - November 2024](https://source.android.com/docs/security/bulletin/2024-11-01) for more information).
+* The Android WebView has been updated to 130.0.6723.73.
+
+## Removed functionality
+
+Following deprecation in earlier releases, these functionalities have been removed from Anbox Cloud with the 1.24.0 release:
+
+* Support for Ubuntu 20.04 (Focal Fossa)
+* Use of node controller to manage the port forwarding from an instance to expose a service
+
+## Deprecations
+
+< List deprecated features. Provide an alternative, if any, for the users to use instead of the deprecated features >
+
+## Known issues
+
+* Out of band support does not work as expected for Android 14 images because of a change in implicit intents feature in the upstream.<!--AC-2852-->
```

Comment:
@adglkh Would you mind sharing the workaround you have found?

Reply:
The argument above is not accurate. 

I think we can point people to the following page
https://developer.android.com/about/versions/14/behavior-changes-all#pending-broadcasts-queued
Specifically.
```
On Android 14, the system can [place context-registered broadcasts in a queue](https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14) while the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle).
```
This is why the app didn’t receive the broadcast, even though it was sent out by the broadcast-data-channels-status action. This also means that any app or service intended to receive the com.canonical.anbox.BROADCAST_DATA_CHANNELS_STATUS broadcast must remain in the running state, not in a **cached** state. This is why our test case fails: the app wasn’t running at system startup.

So I don't think it's a known issue but a caveat if people use out of band v2 feature in Android 14

Reply:
@adglkh We have two options:

1. Drop the known issue as it is actually a caveat like you mentioned.
2. Include the change under Android section of features and improvements (yes, it is not necessarily a feature or improvement) since something has changed in the upstream behaviour and has caused a ripple effect in Anbox as well. We can list something like:
    > The out of band feature in Anbox Cloud works differently with Android 14 because of changes in upstream. On Android 14, the system can [place context-registered broadcasts in a queue](https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14) while the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle). This means that an app needs to be running for it to receive the broadcast.

WDYT?

Reply:
> @adglkh We have two options:
> 
> 1. Drop the known issue as it is actually a caveat like you mentioned.
> 2. Include the change under Android section of features and improvements (yes, it is not necessarily a feature or improvement) since something has changed in the upstream behaviour and has caused a ripple effect in Anbox as well. We can list something like:
>    > The out of band feature in Anbox Cloud works differently with Android 14 because of changes in upstream. On Android 14, the system can [place context-registered broadcasts in a queue](https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14) while the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle). This means that an app needs to be running for it to receive the broadcast.
> 
> WDYT?

A: That sounds good to me. Let me reword this a bit
```
The out of band v2 feature in Anbox Cloud requires the Android app to be running to receive the broadcast If Android 14 image is in use due to the changed behavior that the system [place context-registered broadcasts in a queue]((https://developer.android.com/develop/background-work/background-tasks/broadcasts#android-14)) while the app is in the [cached state](https://developer.android.com/guide/components/activities/process-lifecycle).
```
Please adjust this when needed.